bundles/asdi/asdi-logo-plugin/instance.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @class Oskari.asdi.login.BundleInstance
+ */
+Oskari.clazz.define("Oskari.asdi.logo.BundleInstance",
+    function() {
+      this.dialog = null;
+      this.data = null;
+      this._loc;
+    }, {
+        __name : 'asdi-logo-plugin',
+        getName : function () {
+            return this.__name;
+        },
+
+        start: function() {
+          this._loc = Oskari.getLocalization('asdi-logo-plugin', Oskari.getLang());
+          this.getAboutLinkContent();
+          this.createLogoPlugin();
+        },
+
+        getAboutLinkContent: function() {
+          var me = this;
+          jQuery.ajax({
+            url:Oskari.getSandbox().getAjaxUrl('GetArticlesByTag'),
+            data: {
+              tags: 'asdi-about'
+            },
+            success: function (response) {
+              me.data = response;
+            },
+            error: function (jqXHR, textStatus) {
+                if(typeof errorCb === 'function' && jqXHR.status !== 0) {
+                    errorCb(jqXHR, textStatus);
+                }
+            }
+          });
+        },
+
+        createLogoPlugin(response) {
+          var me = this;
+          var logoService = Oskari.getSandbox().getService('Oskari.map.LogoPluginService');
+          var options = {
+            id:'About',
+            callback: function(event) {
+              me.createAboutDialog(event);
+            }
+          }
+          logoService.addLabel(this._loc.title, options);
+        },
+
+        createAboutDialog(event) {
+          var me = this;
+          if(this.dialog) {
+            this.dialog.close(true);
+            this.dialog = null;
+            return;
+          }
+          var content = jQuery('<div></div>');
+          this.data.articles.forEach( function (article) {
```

Comment:
this.data is null by default. Check that it has articles which is an array? Or better yet, break down the XHR result to a single value. No need to loop it here. This becomes:
if(this.data) {
  content.append(data);
}